app/models/type.rb

```diff
@@ -72,9 +72,9 @@ def <=>(type)
     name <=> type.name
   end
 
-  def self.all
-    find(:all, order: 'position')
-  end
+  # def self.all
+  #  find(:all, order: 'position')
+  # end
```

Comment:
I totally understand this, I am just OCD when it comes to such violations, so I'll just point them out for you to remember :smiley_cat: